### DISCUSSION TITLE
TorchMD on GPU

### DISCUSSION THREAD

**Discussion by npiroozan** [torchmd/torchmd#29] [Q&A]
Hello,

I hope you are doing well.

Would TorchMD work on a GPU which only supports SYCL?

Thank you.

**Comment by giadefa** [comment]
It uses PyTorch, so it works everywhere where PyTorch works.

**Reply by npiroozan** [reply]
Splendid, thank you.

**Reply by npiroozan** [reply]
Hello,

I hope you are doing well.  Does TorchMD support FP32 or bfloat16 precision?

Thank you!

**Reply by stefdoerr** [reply]
It should support FP32. But I don't think bfloat16 will work. At least I have never tried it.

**Comment by giadefa** [comment]
Even TF32 creates problems
